locale/ca/get-involved/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Participa
+layout: contribute.hbs
+---
+
+# Fomentem tota classe de contribucions de la comunitat.
+
+La comunitat de Node.js és gran, inclusiva i entusiasmada per permetre al major nombre d'usuaris,
+contribuir de la manera que puguin. Si vols [reportar un error](https://github.com/nodejs/node/issues),
+[ajudar amb la documentació](/en/get-involved/contribute/) o [contribuir amb el codi](/en/get-involved/development/) del projecte, has vingut al lloc indicat. Explora els recursos de la nostra comunitat per trobar una forma d'ajudar:
+
+<hr>
+
+## Discusió
+
+- La [llista d'errors en ](https://github.com/nodejs/node/issues) és el lloc per discutir les característiques del nucli de Node.js.
+
+- Per xatejar en temps real sobre el desenvolupament de Node vagi a `irc.freenode.net` al canal `#node.js` fent servir un [client d'IRC](http://es.wikipedia.org/wiki/Anexo:Clientes_IRC) o connecti's amb el seu navegador al canal usant [WebChat de freenode](http://webchat.freenode.net/?channels=node.js).
+
+- El compte de Twitter oficial de Node.js és  [nodejs](https://twitter.com/nodejs).
+
+- [Node Weekly](http://nodeweekly.com) és una llista de correu que recopila els últims esdeveniments i notícies al voltant de la comunitat de Node.js.
+
+- [Planet Node](http://planetnodejs.com) és un agregador de blogs de desenvolupadors de Node.
+
+- [NodeUp](http://nodeup.com) és un podcast cobrint les últimes notícies en la comunitat de Node.
+
+- [NodeJS Reactions](http://nodejsreactions.tumblr.com) captura l'experiència de Node.js en forma de GIFs animats.
+
+
+## Aprenentatge
+
+- La [Documentació oficial de l'API](/api) detalla l'API de Node.
+
+- [NodeSchool.io](http://nodeschool.io) li ensenyarà conceptes de Node.js de forma interactiva mitjançant jocs utilitzant la línia de comandos.
+
+- L'[etiqueta de Node.js en StackOverflow](http://stackoverflow.com/questions/tagged/node.js) col·lecciona nova informació cada dia.
+
+- [docs.nodejitsu.com](http://docs.nodejitsu.com/) soluciona molts dels problemes més comuns que la gent es troba.
+
+- [How To Node](http://howtonode.org/) té un nombre creixent d'útils tutorials.
+
+
+## Llocs de la comunitat internaciona i projectes
```

Comment:
internacional*

Reply:
All changes applied. Thanks a lot for the review @jose-lara :D